packages/cli/CHANGELOG.md

```diff
@@ -13,6 +13,7 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 ## [4.13.0] - 2024-06-12
 ### Changed
 - Improve logging and error handling for multichain deploy (#2434)
+- Deprecated flare SDK (#2428)
```

Comment:
This is under an already released version. 

It should also come under `removed`

Reply:
OK，Already updated